common/src/api/external/mod.rs

```diff
@@ -1289,7 +1294,8 @@ impl JsonSchema for Ipv6Net {
                         r#"^([fF][dD])[0-9a-fA-F]{2}:("#,
                         r#"([0-9a-fA-F]{1,4}:){6}[0-9a-fA-F]{1,4}"#,
                         r#"|([0-9a-fA-F]{1,4}:){1,6}:)"#,
-                        r#"\/([1-9]|[1-9][0-9]|1[0-1][0-9]|12[0-8])$"#,
+                        r#"(([0-9a-fA-F]{1,4}){1,6})?"#,
```

Comment:
Update to remove the `{1,6}` bit. Before this would only recognize pure prefixes like `fd00:99::/64` now it can recognize addresses with prefix lengths like `fd00:99::1/64`.